src/HotChocolate/Core/src/Types.Mutations/MutationAttribute.cs

```diff
@@ -0,0 +1,64 @@
+namespace HotChocolate.Types;
+
+public class MutationAttribute : ObjectFieldDescriptorAttribute
+{
+    /// <summary>
+    /// The type name of the argument of the field
+    /// <code>
+    /// type Mutation {
+    ///   createUser(input: ThisIsTheTypeName): CreateUserPayload
+    /// }
+    /// </code>
+    /// </summary>
+    public string? InputTypeName { get; set; }
+
+    /// <summary>
+    /// The name of the argument of the field
+    /// <code>
+    /// type Mutation {
+    ///   createUser(thisIsTheArgumentName: CreateUserCustomInput): CreateUserPayload
+    /// }
+    /// </code>
+    /// </summary>
+    public string? InputArgumentName { get; set; }
+
+    /// <summary>
+    /// The type name of the field in the payload
+    /// <code>
+    /// type ThisIsTheTypeName {
+    ///    user: User
+    /// }
+    /// </code>
+    /// </summary>
+    public string? PayloadTypeName { get; set; }
+
+    /// <summary>
+    /// The name of the field in the payload
+    /// <code>
+    /// type CreateUserPayload {
+    ///    thisIsTheFieldName: User
+    /// }
+    /// </code>
+    /// </summary>
+    public string? PayloadFieldName { get; set; }
+
+    public bool Enabled { get; set; } = true;
```

Comment:
Still not happy with this property name.

Reply:
Disabled = false?

Reply:
or maybe even [DisableMutationConvetion]

Reply:
I changed it to `Disable`.